modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/store/actions/handleFieldRepeated.es.js

```diff
@@ -36,6 +40,7 @@ export default (pages, name) => {
 			const indexToAddField = currentIndex + 1;
 			const newField = {
 				...field,
+				instanceId: generateInstanceId,
```

Comment:
hi, @marcelabc , what do you think about changing "generateInstanceId"  from properties to method ?